src/components/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoListItem from './TodoListItem';
+
+const TodoList = ({
+  todoList, onRemoveTodo, isChecked, handleChange,
+}) => {
+  
+  return (
+    <ul>
+      {todoList.map((todo) => (
+        <TodoListItem
+          key={todo.id}
+          todo={todo}
+          onRemoveTodo={onRemoveTodo}
+          isChecked={isChecked[todo.id]}
+          handleChange={() => handleChange(todo.id)}
+        />
+      ))}
+    </ul>
+  );
+}
+
+TodoList.propTypes = {
+  todoList: PropTypes.array.isRequired,
```

Comment:
for higher strictness, `PropTypes.arrayOf()` will let us specify what types of values are allowed/required in the array: 

```suggestion
  todoList: PropTypes.arrayOf(PropTypes.object).isRequired,
```

you can get really into specifying the exact shape of the object as well, but imo it's too much hassle and another thing where it's better to just wait for Typescript for perfect strictness 😅

Reply:
I can't wait to learn Typescript someday! Changes made.